ifs/ice_effective_radius.F90

```diff
@@ -185,8 +211,10 @@ SUBROUTINE ICE_EFFECTIVE_RADIUS &
     ENDDO
   ENDDO
   !$ACC END PARALLEL
+  !$OMP END TARGET TEAMS DISTRIBUTE PARALLEL DO
 
   !$ACC END DATA
+  !$OMP TARGET EXIT DATA MAP(DELETE:ZMIN_DIAMETER_UM)
```

Comment:
Just occurs to me here (randomly): Since these are standalone directives and not the "closing brackets" of a region, do we not need also `IF(LLACC)` on the exit data directives?

Reply:
As discussed offline, I don't think we need the additional if (LLACC) on the exit data clauses.